applications/conceptual-model-editor/src/diagram/diagram-api.tsx

```diff
@@ -204,11 +204,14 @@ export interface DiagramActions {
   openGroupMenu(groupIdentifier: string, canvasPosition: Position): void;
 
   /**
-   * Sets correct highlighting values in context. We have to call it through the diagram API,
+   * Sets correct highlighting values in context.
+   * We have to call it through the diagram API,
    * because we have access to the rendering library (reactflow) only in diagram component.
-   * @param nodeIdentifier is the identifier of the node to highlight
+   * @param nodeIdentifiers are the identifiers of the nodes to highlight.
```

Comment:
I belive it is not necessary to explain that an array represents multiple things. 
This should be explained and described in a documentation somewhere.